testutil/keeper/session.go

```diff
@@ -152,6 +153,20 @@ func SessionKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 	// Initialize params
 	k.SetParams(ctx, types.DefaultParams())
 
+	// Store block hashes to be used in tests
+	blockHash := map[int64]string{
```

Comment:
1. Exactly, I'm just targeting the blocks being tested in `session_hydrator_test.go`
2. Since `session_hydrator_test.go` is having hardcoded expected session IDs, it would imply changing the way tests are getting their sessionIDs.

Reply:
Can you #PUC with a TODO_IMPROVE. Capture things like:

1. In prod, we'll actually store every block
2. The reason it's not random is because of the session IDs
3. Etc...